modules/dxp/apps/document-library-opener/document-library-opener-onedrive-web/src/main/java/com/liferay/document/library/opener/onedrive/web/internal/service/DLOpenerOneDriveDLAppServiceWrapper.java

```diff
@@ -171,16 +171,16 @@ public FileEntry updateFileEntryAndCheckIn(
 
 			return super.updateFileEntryAndCheckIn(
 				fileEntryId, sourceFileName, mimeType, title, urlTitle,
-				description, changeLog, dlVersionNumberIncrease, null,
-				expirationDate, revisionDate, serviceContext);
+				description, changeLog, dlVersionNumberIncrease, inputStream,
+				size, expirationDate, revisionDate, serviceContext);
 		}
 
 		checkInFileEntry(
 			fileEntryId, dlVersionNumberIncrease, changeLog, serviceContext);
 
 		return super.updateFileEntry(
 			fileEntryId, sourceFileName, mimeType, title, urlTitle, description,
-			changeLog, dlVersionNumberIncrease, null, 0, expirationDate,
+			changeLog, dlVersionNumberIncrease, inputStream, size, expirationDate,
```

Comment:
SF Failing here :D can you Source Format it?

Reply:
Yah, Source Formatted  :D

Reply:
thanks!! :)